src/main/java/com/greenfox/dramacsoport/petclinicbackend/controllers/appUser/auth/RegisterController.java

```diff
@@ -3,42 +3,36 @@
 import com.greenfox.dramacsoport.petclinicbackend.dtos.register.RegisterRequestDTO;
 import com.greenfox.dramacsoport.petclinicbackend.services.appUser.auth.AuthService;
 import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.validation.BindingResult;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.ResponseBody;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
+@RequiredArgsConstructor
+@RequestMapping("api/v1/auth")
 public class RegisterController {
 
-    @Autowired
-    private AuthService authService;
+    private final AuthService authService;
 
     private final Logger logger = LoggerFactory.getLogger(RegisterController.class);
 
+    @SneakyThrows
     @PostMapping("/register")
-    @ResponseBody
     public ResponseEntity<?> registerUser(@Valid @RequestBody RegisterRequestDTO newUserDTO,
                                           BindingResult bindingResult) {
         if (bindingResult.hasErrors()) {
             logger.error("errors during validation {}", bindingResult.getAllErrors());
             return new ResponseEntity<>(bindingResult.getAllErrors(), HttpStatus.BAD_REQUEST);
         }
```

Comment:
If we are here (I know you did not touch this part, but) you should - potentially make this error handling in the ControllerAdvice too.